.github/workflows/publish-nuget.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish NuGet Packages
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  package-nuget:
+    runs-on: windows-latest
+    if: ${{ github.repository == 'twbs/bootstrap' && startsWith(github.event.release.tag_name, 'v') }}
```

Comment:
I'm not exactly sure how NuGet versioning works, so I wanted to mention this. If we create a `v6.0.0-alpha1` version, it will trigger the workflow and produce `v6.0.0-alpha1` NuGet packages.

Do you know if NuGet treats that as the latest version?

If it does, maybe `startsWith` isn't enough: we might need to check that the version matches exactly `vX.Y.Z` without extra characters 🤷